worldengine/generation.py

```diff
@@ -134,11 +134,33 @@ def initialize_ocean_and_thresholds(world, ocean_level=1.0):
             ('plain', hl),
             ('hill', ml),
             ('mountain', None)]
+    armonize_ocean(ocean, e, ocean_level)
     world.set_ocean(ocean)
     world.set_elevation(e, e_th)
     world.sea_depth = sea_depth(world, ocean_level)
 
 
+def armonize_ocean(ocean, elevation, ocean_level):
+    """
+    The goal of this function is to make the ocean floor less noisy.
+    The underwater erosion should cause the ocean floor to be more uniform
+    """
+    width = len(elevation[0])
+    height = len(elevation)
+
+    shallow_sea = ocean_level * 0.85
+    midpoint = shallow_sea / 2.0
+
+    for y in range(height):
+        for x in range(width):
+            e = elevation[y][x]
```

Comment:
Even though it is not that important, I chose to make use of ```[y, x]``` wherever possible - on one hand it is a bit faster (according to tests ~30%, which of course doesn't matter when compared to the whole generation process) and on the other hand I like its indicational effect since it is immediately visible that elevation has already been converted to numpy and thus can be used accordingly. (There probably is a numpy-way to do this loop, too. So maybe add a TODO-comment for later?)

The same goes for line 162.